README.md

```diff
@@ -2,6 +2,19 @@
 
 The all-in-one DataHack management platform, from registration to communications and voting!
 
+## Running the development environment
+1. Setup the project by running `./scripts/setup`
+2. Migrate the database schemas by running `./scripts/migration apply latest`
+3. If you want to populate the database then run `./scripts/backup restore lastest`
+
+> Note: if you want to populate the database from a different version/file run `./scripts/backup restore <file path>`
+
+## Deploying the project
+1. Deploy project by running `./scripts/deploy`
```

Comment:
Please add a step to ssh into the server.

Reply:
Done!